src/stories/Blocks/autosuggest/Autosuggest.stories.tsx

```diff
@@ -25,42 +25,9 @@ export default {
 const Template: ComponentStory<typeof Autosuggest> = (
   args: AutosuggestProps
 ) => (
-  // Apart from <Autosuggest />, everything else is here just for the story
-  // context. The autosuggest styling is directly dependent on the header component,
-  // Please keep up to date with Header.tsx
-  <header className="header" style={{ height: "144px" }}>
-    <div className="header__logo-desktop">
-      <p className="text-body-medium-regular">Context</p>
-    </div>
-    <div className="header__menu">
-      <nav className="header__menu-first">
-        <p className="text-body-medium-regular">Context</p>
-      </nav>
-      <div className="header__menu-second">
-        <div className="header__menu-search">
-          <input
-            className="header__menu-search-input text-body-medium-regular"
-            type="text"
-            placeholder="This field is here just for context."
-          />
-          <img
-            className="header__menu-search-icon"
-            src="icons/basic/icon-search.svg"
-            alt="search icon"
-          />
-          <img
-            className="header__menu-dropdown-icon"
-            src="icons/collection/ExpandMore.svg"
-            alt="expand dropdown icon"
-          />
-          <Autosuggest {...args} />
-        </div>
-      </div>
-    </div>
-    <div className="header__clock">
-      <p className="text-body-medium-regular">Context</p>
-    </div>
-  </header>
+  <div style={{ position: "relative" }}>
```

Comment:
Is there a specific reason to use inline style? I think this is something we would like to avoid.

Reply:
I get what you're saying, but yes, it is necessary.
I've added a comment to make it more clear